docs/core/runtime-discovery/troubleshoot-app-launch.md

```diff
@@ -76,11 +76,11 @@ Every version of .NET offers three different runtime downloads. The following ta
 | .NET Desktop Runtime | Microsoft.NETCore.App<br/>Microsoft.WindowsDesktop.App |
 | .NET Runtime         | Microsoft.NETCore.App |
 
-To install a compatible framework, on the download page for the required .NET version, find a runtime download containing the missing framework. Once you have found the appropriate runtime download, you can then either install it using an [installer](#run-an-installer) or [download the binaries](#download-binaries) and extract them to the desired location.
+To install a compatible framework, on the download page for the required .NET version, find a runtime download containing the missing framework. Once you have found the appropriate runtime download, you can then either install it using an [installer](#run-an-installer) or [script](#run-the-dotnet-install-script). You can also [download the binaries](#download-binaries) and extract them to the desired location.
 
 #### Run an installer
 
-If your existing installation of .NET was installed through an installer or package manager, then also installing the required framework through an installer or package manager is likely the simpler option. Otherwise, you can [download binaries](#download-binaries) instead of using an installer.
+If your existing installation of .NET was installed through an installer or package manager, then also installing the required framework through an installer or package manager is likely the simpler option. Otherwise, you can [run the install script](#run-the-dotnet-install-script) or [download the binaries](#download-binaries) instead of using an installer.
```

Comment:
```suggestion
If your existing installation of .NET was installed through an installer or package manager, then also installing the required framework through an installer or package manager is the recommended option. Otherwise, you can [run the install script](#run-the-dotnet-install-script) or [download the binaries](#download-binaries) instead of using an installer.
```

Using the scripts to modify default install location is typically a bad idea - the installer/package manager will get confused and uninstall or servicing may not work as expected. So we should make it clear that people should prefer the installers as much as possible - and only use the scripts when running from custom locations.